sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1271,10 +1271,11 @@ class DataFrame private[sql](
    * @since 1.6.0
    */
   @scala.annotation.varargs
-  def drop(colNames: String*): DataFrame = {
+  def drop(colName: String, colNames: String*): DataFrame = {
```

Comment:
Should I simply add another `colName: String` if I got you correctly and the problem is due to the ambiguity?